client/src/components/Istoric.component.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import { FaAngleRight, FaAngleLeft } from 'react-icons/fa';
+
+interface CarouselItem {
+  photo: string;
+  text: string;
+  name: string;
+}
+
+interface CarouselProps {
+  items: CarouselItem[];
+}
+
+const Carousel: React.FC<CarouselProps> = ({ items }) => {
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const [timer, setTimer] = useState<number>(0);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setTimer((prevTimer) => prevTimer + 1000); 
+    }, 1000); 
+
+    return () => clearInterval(interval);
+  }, []);
+
+  useEffect(() => {
+    if (timer >= 4500 && timer <= 5000) {
+      setCurrentIndex((prevIndex) => (prevIndex + 1) % items.length);
+    }
+    if (timer >= 7000) {
+      setTimer(0);
+    }
+  }, [timer, items.length]);
+
+  const goToNextSlide = () => {
+    setCurrentIndex((prevIndex) => (prevIndex + 1) % items.length);
+    setTimer(0); 
+  };
+
+  const goToPrevSlide = () => {
+    setCurrentIndex((prevIndex) => (prevIndex - 1 + items.length) % items.length);
+    setTimer(0); 
+  };
+
+  return (
```

Comment:
put md and default value for responsive elements 

https://tailwindcss.com/docs/responsive-design
https://medium.com/@ritikkhndelwal/mastering-responsive-web-design-with-tailwind-css-a-comprehensive-guide-348a843ad948

Reply:
So for small devices like phone screen I should use unprefixed utilities? and if so what it's missing because it looks bad on phone side. On medium side looks good. Thanks in advance.